example/cr-detailed.yaml

```diff
@@ -85,11 +31,11 @@ spec:
     segmentStoreReplicas: 3
     segmentStoreResources:
       requests:
-        memory: "3Gi"
-        cpu: "1000m"
-      limits:
-        memory: "5Gi"
+        memory: "4Gi"
         cpu: "2000m"
+      limits:
```

Comment:
Why are we changing these values? Are these the bare minimum required to deploy pravega?

Reply:
We are modifying these values since we need to ensure the following :-
```
POD_MEM_LIMIT = JVM Heap + Direct Memory
Direct Memory = pravegaservice.cacheMaxSize + 1GB
```